main.go

```diff
@@ -52,76 +55,120 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var kubeAPIServer string
+	var kubeconfig string
+	var watchNamespace string
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&kubeAPIServer, "kubernetes", "", "The Kubernetes API URL, used for in-cluster client construction.")
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file.")
+	flag.StringVar(&watchNamespace, "namespace", "", "Namespace that the controller watches to reconcile objects.")
```

Comment:
Can you name this `kube-namespace` to be inline with tinkerbell/tink#622 and tinkerbell/hegel#106

Reply:
Sure!